python/ray/workflow/tests/test_virtual_actor_2.py

```diff
@@ -305,6 +305,9 @@ def __setstate__(self, n):
     # guarentee obj1's finish.
     obj1 = c.incr.run_async(10)  # noqa
     obj2 = c.incr.run(10)  # noqa
+    # TODO(suquark): The test is flaky sometimes (only on CI), which might indicates
+    # some bugs. This is a workaroundde temporarily.
```

Comment:
Do you want to file an issue and fix this?

Reply:
The related issue: #23760